### PR TITLE
Merge Release Branch so we can fixup stuff in master

### PR DESCRIPTION
**Pull request by micbar**

### CODE REVIEW

File: changelog/unreleased/36856 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Bugfix: Pass OCS-APIREQUEST header on getting shared secret for federation
```
Comment (phil-davis) [inline changelog/unreleased/36856]: changelog needs to be processed out of `unreleased`

File: changelog/unreleased/36546 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Bugfix: stream_read not returning requested length for encrypted remote storage
```
Comment (phil-davis) [inline changelog/unreleased/36546]: changelog needs to be processed out of `unreleased`

Comment (micbar) [reply changelog/unreleased/36546]: will do that after the last PR merged to the release branch.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
I restarted drone - some of the pipelines had hung.
(that has been happening quite a lot today)

**Pull request merged**